ProjectShop/order/models.py

```diff
@@ -68,7 +67,7 @@ class PaymentStatus(models.TextChoices):
 class OrderItemsModel(models.Model):
     order = models.ForeignKey(OrderModel, on_delete=models.CASCADE)
     order_items_qantity = models.PositiveIntegerField(verbose_name=_("Quantity"))
```

Comment:
quantity, maybe

Reply:
Maybe, but it is not my model so I truly don`t think I should change this